src/python/pants/option/global_options.py

```diff
@@ -382,8 +382,10 @@ def register_bootstrap_options(cls, register):
             metavar="<path>",
             daemon=False,
             default=["/etc/pantsrc", "~/.pants.rc"],
-            help="Override config with values from these files. "
-            "Later files override earlier ones.",
+            help=(
+                "Override config with values from these files, using syntax matching that of "
```

Comment:
The syntax of --pants-config-files can be mixed just fine since they must have an extension (you could use 1 ini and 3 toml in that list say) so this new help message seems misleading. I think its trying to help too much and potentially mixing in confusion doing so. Perhaps drop this bit of the change since it should go away anyhow when we're back to only supporting one format.

Reply:
I think that this bit of help is important, particularly since we won't have a suffix on the file. For a long time I didn't actually know what syntax rc files used, because I didn't have one. It seems reasonable to explain "it's the same syntax as that thing over there". I didn't explicitly mention INI or TOML here since that felt ripe to go out of sync post-deprecation.

Reply:
OK, I think explicitly stating INI or TOML would be better than this language even though it requires a cleanup after INI removal because if the current language is left it will be true but obtuse. Now I have to refer to --pants-config-files if I have never set it and just use the default pants.toml because, who knows what that option is and --pants-config-files may be in some format I don't know.

Reply:
+1 to John's suggestion.